cms/templates/js/highlights-editor.underscore

```diff
@@ -2,25 +2,34 @@
 <h3 class="modal-section-title" id="highlights_label"><%- gettext('Section Highlights') %></h3>
 <div class="modal-section-content block-highlights">
     <div role="group" class="list-fields" aria-labelledby="highlights_label">
-        <p class='field-message highlights-description' id='highlights_description'>
-            <%- gettext('Please enter 3-5 highlights to be sent as separate bullet points in the message.') %>
```

Comment:
Hmm.. Should the new modal text go here instead of in the Introduction message?  So it's not faded?

Reply:
If it goes here then it will go underneath the "Section Highlights" header which I think looks weird. I agree that the faded text is hard to read though, so I made it darker. Do you think this looks okay?

![screenshot-20171101154635-766x414](https://user-images.githubusercontent.com/1505923/32294295-1724c0f8-bf1c-11e7-8504-653659634075.png)

Reply:
We discussed this out of band - and realized that the Publish-modal (which also uses the same base modal) has also darkened the intro text.  So we're gonna go with the darker shade.  @thallada will update the SCSS for the base modal so it's consistent across.